### PR TITLE
tests: fix a missing parameter in command to wait for device

### PR DESCRIPTION
**Pull request by valentindavid**

### CODE REVIEW

**Review by MiguelPires — APPROVED** [review]
LGTM

**Review by mardy — APPROVED** [review]
Good catch!

File: tests/lib/nested.sh [hunk 1]
```diff
@@ -951,7 +951,7 @@ nested_add_file_to_vm() {
     ubuntuSeedDev="${devloop}p2"
 
     # Wait for the partition to show up
-    retry -n 2 --wait test -b "${ubuntuSeedDev}" || true
+    retry -n 2 --wait 1 test -b "${ubuntuSeedDev}" || true
```
Comment (sergiocazzolato) [inline tests/lib/nested.sh:954]: should we remove the || true? I mean, this could hide errors like this one.

**Review by sergiocazzolato — COMMENTED** [review]
Nice catch, commented inline

File: tests/lib/nested.sh (hunk 1 above)
Comment (valentindavid) [reply tests/lib/nested.sh:954]: No, because if it is fails, we have a fall back right after.

Comment (valentindavid) [reply tests/lib/nested.sh:954]: So in theory, it can fail. This is because losetup has a race bug which fix is not released yet.

**Review by sergiocazzolato — APPROVED** [review]

**Pull request merged**